Missing module: `pretty-format`

I just tried to run `vue-styleguidist build` and got the following error:

```
module.js:529
    throw err;
    ^

Error: Cannot find module 'pretty-format'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/dan.rumney/IdeaProjects/diagnostics-portal/client/node_modules/vue-styleguidist/bin/styleguidist.js:9:22)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
error Command failed with exit code 1.
```
I'm on Mac OSX, running node v8.6.0 and vue-styleguidist v1.2.8

The offending line appears to be

```
const prettyFormat = require('pretty-format');
```

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
     "lowercase-keys": "^1.0.0",
     "markdown-to-jsx": "^5.4.2",
     "minimist": "^1.2.0",
+    "pretty-format": "^20.0.3",
     "prop-types": "^15.5.10",
     "react": "^15.6.1",
     "q-i": "^1.1.4",

